app/src/lib/ui/ProfessionalList/List.svelte

```diff
@@ -23,7 +23,22 @@
 	const deleteAccount = mutation(deleteAccountMutation);
 
 	async function removeProfessional(professional: Professional) {
-		await deleteAccount({ accountId: professional.account.id });
+		const dateObj = new Date();
+		const month = dateObj.getUTCMonth() + 1;
+		const day = dateObj.getUTCDate();
+		const year = dateObj.getUTCFullYear();
+		const hours = dateObj.getHours();
+		const minutes = dateObj.getMinutes();
+		const suffix = '.deleted' + day + month + year + hours + minutes;
```

Comment:
Je trouverais ça plus logique d'avoir année/mois/jour plutôt que jour/mois/année.
```suggestion
		const suffix = '.deleted' + year + month + day + hours + minutes;
```

Mais sinon, pourquoi ne pas faire tout simplement :
```suggestion
		const suffix = '.deleted' + dateObj.toISOString();
```